src/battle_util.c

```diff
@@ -7954,7 +7954,7 @@ u8 ItemBattleEffects(u8 caseID, u32 battler, bool32 moveTurn)
             }
             break;
         case HOLD_EFFECT_STICKY_BARB:   // Not an orb per se, but similar effect, and needs to NOT activate with pickpocket
-            if (battlerAbility != ABILITY_MAGIC_GUARD)
+            if (IsBattlerAlive(battler) && battlerAbility != ABILITY_MAGIC_GUARD)
```

Comment:
```suggestion
            if (battlerAbility != ABILITY_MAGIC_GUARD)
```

Reply:
Ghoul, I hope it's fine by you if I made the change to merge the PR :sweat_smile: